tctl.yaml

```
@@ -34,14 +34,33 @@ pipeline:
       # GHSA-m425-mq94-257g
       go get google.golang.org/grpc@v1.56.3
 
+      # CVE-2023-47108
+      # go get go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.46.0
+
       go mod tidy
 
       make build
       install -Dm755 tctl ${{targets.destdir}}/usr/bin/tctl
-      install -Dm755 tctl-authorization-plugin ${{targets.destdir}}/usr/bin/tctl-authorization-plugin
+      install -Dm755 tctl-authorization-plugin ${{targets.subpkgdir}}/usr/bin/tctl-authorization-plugin
```

```suggestion
      install -Dm755 tctl-authorization-plugin ${{targets.contextdir}}/usr/bin/tctl-authorization-plugin
```
the main package pipeline needs to use `${{targets.destdir}}` or we've recently added `${{targets.contextdir}}` which can be used in both the main package and sub package pipelines.